README.md

```diff
@@ -137,6 +137,9 @@ Table of Contents
   * [IP2Location](https://www.ip2location.com) — Freemium IP geolocation service. LITE database is available for free download. Import the database in server and perform local query to determine city, coordinates and ISP information.
   * [FraudLabs Pro](https://www.fraudlabspro.com) — Screen an order transaction for credit card payment fraud. This REST API will detects all possible fraud traits based on the input parameters of an order. Free Micro plan has 500 transactions per month.
   * [IPList](https://www.iplist.cc) — Lookup details about any IP address, such as Geo IP information, tor addresses, hostnames and ASN details. Free for personal and business users.
+  * [ClearIP](https://clearip.io) — An embarrassingly simple API that provide3s reliable and useful IP geolocation data for your business. 
```

Comment:
Not clear what the free plan for this one is from their site. 

Mind adding some data about what’s free on both - something we can verify on their site.

Reply:
Currencystack.io has 2k free requests per month. Clearip.io has 50k per month.

Reply:
I saw currencystack free tier on their site but not clearip. It needs to be on the site

Reply:
It's now on the pricing page https://clearip.io/pricing/

Reply:
Hmm, how does one sign up for that? Only sign up options are the paid ones